### DISCUSSION TITLE
Definicion del Proyecto Imposible

### DISCUSSION THREAD

**Discussion by Gona79** [ISPC-TST-Electronica-Microcontrolada/Grupo5#30] [Announcements]
Que tal estimados, les dejo este espacio para que definan su proyecto y lo expliquen brevemente. De esta forma podremos habrir el registro con el profesor Jorge. Haganlo con el siguiente formato: 

Tema del Proyecto: 
Nombre del Proyecto:  
Definicion del Proyecto:

**Comment by JorEl057** [comment]
Chicos, que tal, como están :
Que pasó que no han avanzado en la realización del Proyecto Imposible? Tuvieron algún problema?

**Comment by Mauro-Martinez** [comment]
Hola profe como esta?
La verdad es que tuvimos un pequeño inconveniente a la hora de avanzar con
el proyecto,  se nos dificultó bastante. Pero ya nos pudimos organizar y
definir que tipo de trabajo vamos a realizar, más tardar mañana estaremos
subiendo las consignas al repositorio. Desde ya muchas gracias y disculpe
las demoras ocasionadas :)

**Comment by Mauro-Martinez** [comment]
Tema del Proyecto: Traductor de Lenguaje de Señas.
Nombre del proyecto: Guante de Señas
Definición del Proyecto:Es importante aprender lenguaje de señas, ya que
nos sensibiliza hacia la comunicación e inclusión de las personas sordas y
mudas; para ello realizaremos una simulación de un guante traductor de
lenguaje de señas.
